src/components/05_pages/Permissions/Permissions.js

```diff
@@ -19,12 +19,20 @@ const Permissions = class Permissions extends Component {
           `${
             process.env.REACT_APP_DRUPAL_BASE_URL
           }/admin-api/permissions?_format=json`,
+          {
```

Comment:
Do you mind moving this into its own PR :)

Reply:
Can we leave it for now? I was going to work on a follow up PR that puts all this API stuff into a separate wrapper file as there's a lot of repetition here

Reply:
I'd suggest to not abstract till we have another usecase.

Reply:
I would have suggest @mattgrill to work on it, given he has actually written a library doing exactly that.

Reply:
To clarify, I wanted to express: @mattgrill would be a good candidate when you don't have time to work on it.

Reply:
@justafish @dawehner I don't mind working on this less-than-glamorous part. I was already planning on doing something like this when I do the changes to update Drupal with the permissions changes.

Reply:
Additionally, I think this is fine to add here. I'll probably just copy all this out into it's own helper anyways.

Reply:
:+1: